cypress/integration/tests/administrator/repository/git.test.ts

```diff
@@ -102,25 +101,24 @@ describe("Test an application form a Git source", { tags: "@tier1" }, () => {
         application.openreport();
     });
 
-    it("Disable insecure git Repository", () => {
+    it("Analysis on insecure git Repository(http) for tackle test app", function () {
         GitConfiguration.open();
-        gitConfiguration.toggleInsecureGitRepositories();
-    });
+        gitConfiguration.disableInsecureGitRepositories();
```

Comment:
Put some comments in the method to explain the TC like : 


//negative test case , disable Consume insecure repo (http)repos not allowed and then run analysis. Analysis should fail